modules/mockserver/coverage.out

```diff
@@ -0,0 +1 @@
+mode: set
```

Comment:
Do we need this coverage file here? If so what would be the rationale for it, in the case we need to spread it to the rest of the modules.

Reply:
Nope, sorry, that was generated somehow, let me remove it.